chart/validator/templates/validator-config.yaml

```diff
@@ -5,6 +5,17 @@ kind: ValidatorConfig
 metadata:
   name: validator-config
 spec:
+  helmConfig:
+    registry: {{ required ".Values.helmConfig.registry is required!" .Values.helmConfig.registry }}
+    {{- if .Values.helmConfig.caFile }}
+    caFile: {{ .Values.helmConfig.caFile }}
+    {{- end }}
+    {{- if .Values.helmConfig.insecureSkipVerify }}
```

Comment:
nit: do any of these values need to be wrapped in `ifs`? I think `insecureSkipVerify` definitely doesn't.

Reply:
yeah they arent needed because of `omitempty`